src/Nowruz/general/input/input.tsx

```diff
@@ -0,0 +1,51 @@
+import ErrorOutlineRoundedIcon from '@mui/icons-material/ErrorOutlineRounded';
+
+import css from './input.module.scss';
+import { InputProps } from './input.types';
+
+export const Input = (props: InputProps) => {
```

Comment:
Why didn't you use material ui input

Reply:
I think in some simple components it is not needed to use Material UI since its styling has some complexities